modules/matchPattern.js

```diff
@@ -1,22 +1,20 @@
 import pathToRegexp from 'path-to-regexp'
 
-const cache = {}
+// cache[exactly][pattern] contains getMatcher(pattern, exactly)
+const cache = {true: {}, false: {}}
```

Comment:
Good catch! As a Pythonista, I failed to notice that these keys are strings / symbols, not booleans. I agree that it feels sloppy...

That said, adding explicit type conversions doesn't make the code more readable. Perhaps it's best to live with the implicit conversion.